src/icons/font/index.js

```diff
@@ -53,6 +61,17 @@ const Icons = {
     {name:"sub arrow left", icon: SubArrowLeftIcon},
     {name:"sub arrow right", icon: SubArrowRightIcon},
     {name:"arrow right", icon: ArrowRightIcon}
+  ],
+
+  editor: [
+    {name:"bold", icon: BoldIcon},
+    {name:"code", icon: CodeIcon},
+    {name:"header", icon: HeaderIcon},
+    {name:"italic", icon: ItalicIcon},
+    {name:"list ordered", icon: ListOlIcon},
+    {name:"list unordered", icon: ListUlIcon},
+    {name:"newspaper", icon: NewspaperIcon},
```

Comment:
Material does not have a newspaper icon. Please try to find an icon with a name from the material icons that match.